docs/admin/troubleshooting.md

```diff
@@ -326,112 +326,407 @@ Verify if you're using Infiniband for `fabric_iface`: in the server config. The
 ## Common Errors and Workarounds
 
 ### Use dmg command without daos_admin privilege
+```
+# Error message or timeout after dmg system query
+$ dmg system query
+ERROR: dmg: Unable to load Certificate Data: could not load cert: stat /etc/daos/certs/admin.crt: no such file or directory
+
+# Workaround
+
+# 1. Make sure the admin-host /etc/daos/daos_control.yml is correctly configured.
+	# including:
+		# hostlist: <daos_server_lists>
+		# port: <port_num>
+		# transport\config:
+			# allow_insecure: <true/false>
+			# ca\cert: /etc/daos/certs/daosCA.crt
+			# cert: /etc/daos/certs/admin.crt
+			# key: /etc/daos/certs/admin.key
+
+# 2. Make sure the admin-host allow_insecure mode matches the applicable servers.
+```
+### use daos command before daos_agent started
+```
+$ daos cont create $DAOS_POOL
+daos ERR  src/common/drpc.c:217 unixcomm_connect() Failed to connect to /var/run/daos_agent/daos_agent.sock, errno=2(No such file or directory)
+mgmt ERR  src/mgmt/cli_mgmt.c:222 get_attach_info() failed to connect to /var/run/daos_agent/daos_agent.sock DER_MISC(-1025): 'Miscellaneous error'
+failed to initialize daos: Miscellaneous error (-1025)
 
-	# Error message or timeout after dmg system query
-	$ dmg system query
-	ERROR: dmg: Unable to load Certificate Data: could not load cert: stat /etc/daos/certs/admin.crt: no such file or directory
 
-	# Workaround
+# Work around to check for daos_agent certification and start daos_agent
+	#check for /etc/daos/certs/daosCA.crt, agent.crt and agent.key
+	$ sudo systemctl enable daos_agent.service
+	$ sudo systemctl start daos_agent.service
+```
+### use daos command with invalid or wrong parameters
+```
```

Comment:
do we really want to document this? sounds trivial to me...

Reply:
Same as above it's already in the file and not part of my patch.